<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dev" ignored="3" name="Test(failed)" tests="1" failures="1" timestamp="05 Nov 2017 03:43:19 GMT" time="0.063" errors="0">
  <testcase name="Bb" time="0.019" classname="testNGTests.GroupsTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at testNGTests.GroupsTest.Bb(GroupsTest.java:28)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Bb -->
  <testcase name="Aa" time="0.0" classname="testNGTests.GroupsTest">
    <ignored/>
  </testcase> <!-- Aa -->
  <testcase name="Bbb" time="0.0" classname="testNGTests.GroupsTest">
    <ignored/>
  </testcase> <!-- Bbb -->
  <testcase name="Aaa" time="0.0" classname="testNGTests.GroupsTest">
    <ignored/>
  </testcase> <!-- Aaa -->
</testsuite> <!-- Test(failed) -->
